R/Data-validity.R

```diff
@@ -165,26 +165,17 @@ validate_data_mixture <- function(object) {
 #' contains valid elements with respect to their types, dependency and length.
 validate_data_DA <- function(object) {
   o <- Validate()
+  # In if clause so that below test_* won't fail.
+  if (!test_number(object@Tmax, lower = 0 + .Machine$double.xmin)) {
```

Comment:
unfortunately `test_number` does not allow us to say `strictly greater`. therefore I would here do 
```suggestion
  if (!(test_number(object@Tmax) && object@Tmax > 0)) {
```

Reply:
this is a bit cleaner